modules/ROOT/partials/configuration/dialog_streamContent.adoc

```diff
@@ -0,0 +1,58 @@
+[[streamContent]]
+== `streamContent`
+
+`+streamContent+` is an optional property, available to add to the `+iframe+` dialog component.
+
+The property has a default value of `+false+`.
+
+When `+streamContent+` is set to true, content presented within the `+iframe+` dialog component is updated using the `document.write()` method, rather than the by using the `+srcdoc+` attribute.
+
+The `+srcdoc+` attribute requires the `+iframe+` be reloaded each time the content changes. This results in a flickering effect.
+
+As well, performing updates via the `+srcdoc+` attribute means the contents of the `+iframe+` are not accessible. As a consequence, it is not possible to automatically scroll the `+iframe+` contents to the bottom as it gets updated.
+
+Setting `+streamContent+` to `+true+`, and switching the update mechanism to the `document.write()` method, stops the flickering and allows the `+iframe+` to constantly present the incoming content at an end-of-the-current data position.
+
+*Type:* `+Boolean+`
+
+*Possible values:* `+true+` `+false+`
+
+*Default value:* `false`
+
+=== Example: setting an `iframe` component to display updating content using the `streamContent` property
+
+[source,js]
+----
+tinymce.activeEditor.windowManager.open({
+  title: 'Dialog Title', // The dialog's title - displayed in the dialog header
+  size: 'normal',
+  body: {
+    type: 'panel', // The root body type - a Panel or TabPanel
+    items: [ // A list of panel components
+    {
+      name: 'preview',
+      type: 'iframe',
+      streamContent: 'true',
+      border: 'true',
+    }
+    ]
+    },
+    buttons: [ // A list of footer buttons
+      {
+        type: 'submit',
+        text: 'OK'
+      }
+    ]
+}, { inline: 'bottom' } );
+----
+
+[NOTE]
+.Dialog position
+====
+The example above also adds a second argument — `{ inline: 'bottom' } — to the dialog. This argument changes the presented position of the dialog within the {productname} editor.
+
+For information on using this property see the xref:dialog-configuration.adoc#dialog-position[Dialog position] section of the xref:dialog-configuration.adoc[Dialog configuration documentation].
+====
```

Comment:
@shanmen-tiny Can you confirm that we should be documenting `{ inline: 'bottom' } );`?

My understanding from our conversation was that this argument is **not** something an integrator can change/update, which leads me question if this should be in the docs or kept internally?

Reply:
@kemister85 The API or `windowManager` is exposed to the public, I think we should document it. It is not possible to change the dialogs location or option that we have built in the plugins/core. However, if they wish they can choose to use the API to create custom dialogs and have it positioned/opened at the desired location.